src/Uno.UI/UI/Input/PointerPoint.cs

```diff
@@ -49,7 +50,17 @@ public PointerPoint(Windows.UI.Input.PointerPoint point)
 			Properties = new PointerPointProperties(point.Properties);
 		}
 
-		public static explicit operator Windows.UI.Input.PointerPoint(Microsoft.UI.Input.PointerPoint muxPointerPoint)
+		// Historically, we had explicit conversion only.
+		// In the work for InteractionTracker, we needed an implicit conversion to avoid a breaking change.
+		// The compiler doesn't allow to define both. (https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0557)
+		// And changing the explicit conversion operator to implicit conversion operator is a binary breaking change.
+		// We manually add this method to avoid this binary breaking change.
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public static Windows.UI.Input.PointerPoint op_Explicit(Microsoft.UI.Input.PointerPoint muxPointerPoint)
```

Comment:
Do we have a trace of this to be removed in uno 6 ?

Reply:
https://github.com/unoplatform/uno/issues/15508